src/test/java/com/karankumar/bookproject/backend/repository/CustomShelfRepositoryTest.java

```diff
@@ -19,18 +19,20 @@
 
 import com.karankumar.bookproject.annotations.DataJpaIntegrationTest;
 import com.karankumar.bookproject.backend.entity.CustomShelf;
+import com.karankumar.bookproject.backend.entity.account.User;
+import com.karankumar.bookproject.utils.SecurityTestUtils;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 
-import java.util.List;
-
 import static org.assertj.core.api.Assertions.assertThat;
 
 @DataJpaIntegrationTest
 class CustomShelfRepositoryTest {
+    @Autowired private UserRepository userRepository;
     @Autowired private CustomShelfRepository repository;
     private final String customShelf1 = "Test1";
+    private User user;
 
     @Test
     @DisplayName("When shelf exists, findByShelfName returns one shelf")
```

Comment:
Could you please add test for checking if there are more than one shelves belonging to different users and we return only the ones belong to the correct users?